docs/developer-guide/rules.md

```diff
@@ -186,17 +186,27 @@ If the rule has autofix use:
 
 Depending on the rule, it might be possible to automatically fix the rule's problems by mutating the PostCSS AST (Abstract Syntax Tree) using the [PostCSS API](http://api.postcss.org/).
 
+Add `fixable: true` to the rule's `meta`:
```

Comment:
[suggest] Minor. The following seems a bit more intuitive to me:

```suggestion
Set `meta.fixable = true` to the rule:
```

But it's not a strong opinion. No problems with leaving it as-is. 🙆🏼‍♂️